### PR TITLE
Create Rendering_Element .md

### PR DESCRIPTION
**Pull request by AnkitaGupta454**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes # 2498

<hr>

## Type of change 📝
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] MERN
<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: MERN/Rendering_Element .md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+<div id="root"></div>
+
+```
+<p> ReactDOM manages everything that is inside this div and this root is called as DOM node </p>
+
+<br>
+
+<p> Now, for rendering simple React element to root node, one has to write following code in App.js file </p>
```
Comment (Naman-1234) [inline MERN/Rendering_Element .md:30]: Correction: It is not mandatory to name it App.js, It is just a convention which everyone generally follow. Everything should be inside the file which is taken as a first parameter when using React.render method.

**Review by saloni-15 — CHANGES_REQUESTED** [review]
you have to make changes in the related folder, for react you have to work on `MERN/Topic/Frontend/React`

File: MERN/Rendering_Element .md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  It describes what you want to see on the screen.
+  Example of an element is :-<p>
+
+```
+
+const ele = <h1> Rendering Elements in react </h1>;
+
+```
+
+<p>React DOM update the DOM with the content in the react element. React elements are cheap to create, and are plain objects. Its a simple object that describes DOM node and its attributes.
```
Comment (Naman-1234) [inline MERN/Rendering_Element .md:13]: This part is little confusing.Please try to rephrase it.

File: MERN/Rendering_Element .md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+```
+
+<h3> It produces following output on the screen </h3>
+
+<img width="646" alt="output" src="https://user-images.githubusercontent.com/69414725/134839557-4ded4e80-c154-40e3-90f6-576d8290b253.PNG">
+
+## Updating an element in React
+
+<p> React elements ae immutable i.e, once you create them you can't change its children and attributes. Therefore, if you want to update an element in React, you must use render() method several time
```
Comment (Naman-1234) [inline MERN/Rendering_Element .md:45]: Please also mention the reason why is it required and why we cannot do it without using the render method. Try to throw some light on a concept like Virtual DOM and how React manage it to render element.

**Review by Naman-1234 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**